source/Cleaner.cpp

```diff
@@ -942,7 +865,8 @@ Alignment *Cleaner::cleanSpuriousSeq(float overlapColumn, float minimumOverlap,
     overlapVector = new float[alig->originalNumberOfSequences];
 
     // Compute the overlap's vector using the overlap column's value
-    if (!calculateSpuriousVector(overlapColumn, overlapVector))
+    alig->Statistics->calculateSeqOverlap();
```

Comment:
I added a `calculateSpuriousVector` method to the `Manager` so that it can initialize the `statistics` module if needed, instead of doing it directly in `Cleaner` (will help later with dynamic dispatching). In addition, I updated `calculateSpuriousVector` to call `calculateSeqOverlap` only once.